package/base-files/Makefile

```diff
@@ -66,14 +64,15 @@ define Package/base-files/conffiles
 /etc/iproute2/rt_tables
 /etc/passwd
 /etc/profile
-/etc/profile.d
+/etc/profile.d/
 /etc/protocols
 /etc/rc.local
 /etc/services
 /etc/shadow
 /etc/shells
 /etc/shinit
 /etc/sysctl.conf
+/etc/sysctl.d/
```

Comment:
So, you are now trying to trick this in here, after I've NAKed it in #2996 ? That's not acceptable.

Reply:
> So, you are now trying to trick this in here, after I've NAKed it in #2996 ? That's not acceptable.

You're missing the larger point which is I'm trying to rationalize the presence of both a conffiles in `files/lib/upgrade/keep.d` and embedded in the `Makefile` itself.

If you want the line `/etc/sysctl.d/` removed above that's fine, but you're not seeing the bigger problem from the minutiae if you're closing the PR solely based on that.

Please reopen it.

Reply:
I obviously am aware of the fact the this PR is about something different than adding support for /etc/sysctl.d (although the commit does not contain any explanation). Therefore, adding sysctl.d simply has nothing to do with what you are dealing with in this PR. So, if you add this unrelated change here, I must assume that you did it in order to slip it in (particularly since this was posted _after_ our discussion in the other PR).

Based on that, I cannot trust you anymore, and therefore I'm minimizing the risk of you sneaking in something else.

Reply:
> So, if you add this unrelated change here, I must assume that you did it in order to slip it in (particularly since this was posted after our discussion in the other PR).

How about you don't make assumptions about my character or motivation and I won't do so about your either.

I have that change because *I* need it on my running system (my production Supermicro SYS-5018D-FN4T) and I tested the PR exactly as I'm using it locally.  _Occam's Razor_, mate.

> Based on that, I cannot trust you anymore, and therefore I'm minimizing the risk of you sneaking in something else.

You don't want to approve it, that's fine.  But you don't have the right to close it preemptive either as you're not the sole owner/reviewer.

If you have a problem with a PR, you call it out and you give people the chance to remedy.  That's how this works.

Reply:
> I have that change (in the form you see it) because I need it on my running system (my production Supermicro SYS-5018D-FN4T) and I tested the PR exactly as I'm using it locally. Occam's Razor, mate.

Might be, but this isn't about your device and what _you_ need, but about what's best for OpenWrt.

> You don't want to approve it, that's fine. But you don't have the right to close it preemptive either as you're not the sole owner/reviewer.

Since you included a change which I don't think should go into the repo, I NAKed it. Just because that happens rarely it doesn't mean that it won't happen at all.

> If you have a problem with a PR, you call it out and you give people the chance to remedy. That's how this works.

And you would have had much more success if you weren't so focussed on teaching me about how things work.

After all, you had the chance to admit that you added something which isn't strictly necessary for the change you proposed, and that this might have created a wrong impression. (You wouldn't even have had to admit any bad intentions.) This would have easily rebuilt trust, and convinced me to reopen this again.

However, you didn't try to reach out for that, but tried to tell me what I already knew first and then tried to teach me about what I have to do second. So, I must fear that you will try to get sysctl.d in again when I'm not looking, and that's why I'm reducing that risk by keeping this closed.

Reply:
> Might be, but this isn't about your device and what you need, but about what's best for OpenWrt.

Refreshing to hear you say that.

I believe that what's best for the project is:

1. remembering that OSS works when people keep in mind it's collaborative;
2. giving people the benefit of the doubt;
3. sharing the PR that they actually **tested** and not some chimera that's not been tested;

all of which is what I'm trying to do.

> Just because that happens rarely it doesn't mean that it won't happen at all.

Again, the change you see is the change that's been tested.

Further, PR #2996 hasn't reached a sense of consensus.  Having a strong opinion about it singularly isn't a substitute for the group reaching agreement.

If you read the 2nd comment **carefully**, it's quite clear that the PR doesn't work as intended and it's still a work-in-progress, subject to changes.  No one has represented it any other way.  No one has attempted to trick anyone.

Your requested change, as well as any others required to make this fix work as desired and get it **finalized** will happen as part of the process unless someone gets trigger-happy and decides to shutdown the process prematurely...

> And you would have had much more success if you weren't so focussed on teaching me about how things work.

I'm not the one trying to prematurely squelch discussion here.

> This would have easily rebuilt trust, and convinced me to reopen this again.

As anyone with commit rights for core could reopen this, I was hoping someone reasonable might do just that.

Presumably after they had read all of the comments, and fully grokked the submitter's intent.

Closing the PR doesn't indicate an interest in discussing changes; it indicates a desire to end the conversation without further evolution.

> However, you didn't try to reach out for that, but tried to tell me what I already knew first and then tried to teach me about what I have to do second.

Really?  Because nothing you said before closing this PR seemed to indicate **any** acknowledgement that I clearly called it out as a work-in-progress and not a finished product.

Further, telling me I'm attempting "trickery" is effectively convicting me of a "thought crime".  We try not to be Orwellian here.  It's not explicitly called out in the Best Practices, but it's well understood...

Lastly, closing out a PR without asking any questions not based on rhetorical suppositions of intention is not a good model of "reaching out", if that's the behavior you're trying to model for the less evolved others of us.